### ISSUE TITLE
Little differences between GTK3 and GTK2 widgets

### ISSUE THREAD

**Issue by jEsuSdA** [shimmerproject/Greybird#163]
Hi!

There are some differences between GTK3 and GTK Widgets.

Here, an example (GTK3 (Pluma text editor) at left, GTK2 at right (Leafpad text editor):

![screenshot1](https://cloud.githubusercontent.com/assets/5833446/20437949/1cf671c0-adb6-11e6-869b-43b6b386a504.png)

As you can see, the search text field is higher on GTK3 than GKT2.

Another difference is when you click or select any button: On GTK3 the button goes darker but on GTK2 the button goes Blue.

The checkbox bug still remain in latest Greybird version as I told you in bug 152 (https://github.com/shimmerproject/Greybird/issues/152).

Thank you! ;)

**Comment by ochosi** [comment]
Yes I know, there are several differences and even more prominently the checkboxes themselves look fairly different. I haven't had time/energy to adjust those yet and it will take a while longer until those things match 100% again.
